syntax/gleam.vim

```diff
@@ -8,32 +8,65 @@ let b:current_syntax = "gleam"
 
 " Keywords
 syntax keyword gleamKeyword
-  \ module import pub panic use
-  \ type let as if else todo const
-  \ case assert tuple try opaque
+  \ import pub panic use type let as if else todo const case assert try opaque
```

Comment:
What has changed here? I can't tell due to the unneeded formatting change.

Reply:
Reverted the formatting, sorry about that.
It removes `module` and `tuple` as keywords. I couldn't find them in the treesitter-grammar, so I thought they didn't belong

Reply:
Thank you